bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BookKeeper.java

```diff
@@ -81,6 +81,13 @@ static BookKeeperBuilder newBuilder(final ClientConfiguration clientConfiguratio
      */
     CompletableFuture<LedgerMetadata> getLedgerMetadata(long ledgerId);
 
+    /**
+     * Return driver metadata service is available.
+     *
+     * @return the metadata service is available.
+     */
+    CompletableFuture<Boolean> isDriverMetadataServiceAvailable();
```

Comment:
Add a default implementation to avoid breaking the interface

Reply:
I doubt that add default implementation is a good idea. First, we don't guarantee/make ABI comptabile between minor releases; Second, People whole implement metadata driver should implement this.

cc @eolivelli @dlg99

Reply:
Since it is not a required thing for the bookkeeper. I think it should be ok to implement a default handle. @shoothzj

Reply:
@zymap This default interface might be error prone. I have started a mail thread. I think we can continue discuss in https://lists.apache.org/thread/fk02bz4lggz086kgwwxdw2yv1ktn7x35